### ISSUE TITLE
Add reference to CIPSEA on Data page

### ISSUE THREAD

**Issue by krusynth** [krusynth/digitalpolicy#10]
* [ ] Link: https://en.wikipedia.org/wiki/Confidential_Information_Protection_and_Statistical_Efficiency_Act
* [ ] This would be a good time to build out the E-Government Act page as well.

**Comment by amy-usds** [comment]
I can spend some of my free time on both within the next week or so... not that I have any experience w/ the E-Gov Act.

Speaking of (somewhat related), I noticed Computer Chaos listed somewhere. Does it make sense to compile additional links to major reports?

**Comment by krusynth** [comment]
Ironically, *all* of this project started because I couldn't find a digitized copy of Computer Chaos on the internet! Then I started annotating it, and well here we are. That report is fundamental to all government IT policy that's happened since, though most of the problems reported still exist, so I think it's a load-bearing element.  Happy to add other reports if they make sense in the context.

I've got a pretty good handle on the e-gov act, I just need to spend the time to write it - there are rough notes already.  I'll try to get to that over the long weekend.
